packages/chart/tests/chart.browser.spec.ts

```diff
@@ -33,7 +33,10 @@ describe("@hpcc-js/chart", () => {
         const item = (chart as any)[key];
         if (item) {
             if (!urlSearch || urlSearch === item.prototype.constructor.name) {
-                describe(`${item.prototype.constructor.name}`, () => {
+                describe(`${item.prototype?.constructor?.name}`, () => {
+                    it("Simple", () => {
+                        expect(true).to.be.true;
+                    });
```

Comment:
I noticed a few of these being added (here and in some other spec files) & just didn't understand what they were testing

Reply:
A `describe` with no `it` will cause an error, this ensures there is always at least 1 `it`